docs/api/en/core/BufferAttribute.html

```diff
@@ -63,6 +63,9 @@ <h3>[property:Integer count]</h3>
 			or color), then this will count the number of such vectors stored.
 		</p>
 
+		<h3>[property:Number gpuType]</h3>
+		<p>Configures the internal bound GPU type. Either [page:BufferAttribute THREE.FloatType] or [page:BufferAttribute THREE.IntType], default is [page:BufferAttribute THREE.FloatType].</p>
+
```

Comment:
Open to clarification here. This would have to match the declared type in the vertex shader. https://threejs.org/examples/#webgl2_buffergeometry_attributes_integer would be a reference.